agent/src/main.cpp

```diff
@@ -137,8 +138,8 @@ int main(int argc, char **argv)
     int userID = 0;
     bool keepContainersAlive = false;
     int timeout = -2;
-    Glib::ustring serviceManifestDir = SERVICE_MANIFEST_DIR_OPT;
-    Glib::ustring defaultServiceManifestDir = DEFAULT_SERVICE_MANIFEST_DIR_OPT;
+    Glib::ustring serviceManifestDir = "";
+    Glib::ustring defaultServiceManifestDir = "";
```

Comment:
Would be nice to have a constant that these are set to that can be used to check if they were set later.
Remove magic values.